buildpacks/nodejs-runtime/Cargo.lock

```diff
@@ -0,0 +1,2082 @@
+# This file is automatically @generated by Cargo.
```

Comment:
To speed up compiles and for better UX, I'd recommend using the Cargo workspaces feature, where all of the crates in this repo will share a single `target/` directory and `Cargo.lock` (they'll still have separate `Cargo.toml` files). 

That way all of the dependencies will only have to be compiled once per workspace, not once per crate. It also allows running cargo commands from the repo root, rather than always having to be in the correct crate directory (running from an individual crate still works).

https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html
https://doc.rust-lang.org/cargo/reference/workspaces.html

Reply:
Yeah, this was on my list of like 100 things to do in this branch. Done in fa66. It's definitely a little better with workspaces (though `cargo libcnb package` is a little weird).